syntax = "proto3";
import "vectorization_config.proto";
import "label_config.proto";

message FrameConfig {
  uint64 window_width_ms = 1;
  uint64 bucket_width_ms = 2;
  repeated VectorizationConfig vectorization_configs = 3;
  repeated LabelConfig label_configs = 4;
}
